Events/index.md

```diff
@@ -6,10 +6,14 @@ author: Doug, Jackson, & Tony
 
 # Upcoming Conferences
 * 17 February 2017 - Oregon State University [College of Engineering Graduate Research Showcase](http://engineering.oregonstate.edu/graduate-research-showcase-student-information-and-registration)
-  - Poster due 7 Feb
-* 11-15 June 2017 - American Nuclear Society Annual Meeting ([ANS](http://ansannual.org))
-  - Summary due 1 Feb
+  - Poster due 7 Feb.
+* 11-15 June 2017 - [2017 American Nuclear Society Annual Meeting](http://ansannual.org)
+  - Summary due 8 Feb.
```

Comment:
hah! if it gets pushed another week maybe! 

(then again I've been saying that since the original deadline)